sycl/doc/GetStartedGuide.md

```diff
@@ -179,7 +179,7 @@ CUDA backend has Windows support; Windows Subsystem for Linux (WSL) is not
 needed to build and run the CUDA backend.
 
 Enabling this flag requires an installation of at least
-[CUDA 10.2](https://developer.nvidia.com/cuda-10.2-download-archive) on
+[CUDA 11.5](https://developer.nvidia.com/cuda-11-5-0-download-archive) on
```

Comment:
Can we change this to 11.0? Our plugins are aimed to target 11.0 so would be good if we had the same support in open source

Reply:
Yeah is it definitely working now for 11.0?

Reply:
Pending https://github.com/intel/llvm/pull/10922/files it should be

Reply:
Will it work for e.g. 10.2 or even earlier? Ideally we can support as many cuda versions as possible. It's been pointed out many times that clusters don't update their versions for long periods e.g. https://github.com/intel/llvm/issues/10858#issue-1855051343

Reply:
I think as long as we keep intermittently checking that CUDA 10.2 is working we can keep this support. But the problem is that we don't have any CI for CUDA 10.2 whereas we do have CI for CUDA 11.0 (to test the releases internally)

Reply:
I can just say this in the doc "we don't have any regular testing for CUDA < 11 but all version down to 10.2 should work".

Do you remember why 10.2 was the lowest (I forgot). Just want to check we can't go lower.

Reply:
There are version checks for above/below 10.2 here: https://github.com/intel/llvm/blob/f69dae7972062d90f804c65996d31f246d2288b3/sycl/plugins/unified_runtime/ur/adapters/cuda/usm.cpp#L242

Maybe with the right checks it < 10.2 could work fine. Or maybe it already works.

Reply:
I would say that it already works. But we don't have CI to check

Reply:
intel/llvm CI started with 11.4 version.
We switched to 11.7 due to build issues of SYCL-CTS. https://github.com/intel/llvm/pull/6545
Today we validate each commit with 12.1 version, but I unfortunately, I didn't record the reason for uplift from 11.7.

> I would say that it already works. But we don't have CI to check every commit

Do we need to use 10.2 in CI instead of 12.1?
IIRC, all previous uplifts were agreed with @AerialMantis.

Reply:
I believe that from the next release we are meant to be supporting 12.1, so it makes sense the CI tests that. It would be nice to keep track of whether older releases still work, so that systems that don't regularly update their toolkit can still stay supported; but I think that is more on is to test from time to time and when introducing new features that might break older versions.